DotNet/Census/Domain/Managers/CensusPatientListManager.cs

```diff
@@ -74,6 +78,29 @@ public async Task<IEnumerable<CensusPatientListEntity>> GetPatientList(string fa
         }
     }
 
+    public async Task<CensusCount> GetCensusCount(string facilityId, DateTime? startDate, DateTime? endDate,
+        CancellationToken cancellationToken = default)
+    {
+        List<CensusPatientListEntity> patients;
+        
+        if (startDate.HasValue && endDate.HasValue && startDate.Value != default && endDate.Value != default)
+        {
+            patients = (await _patientListRepository.FindAsync(c => c.FacilityId == facilityId && c.AdmitDate >= startDate && c.AdmitDate <= endDate, cancellationToken)).DistinctBy(p => p.PatientId).ToList();
+        }
+        else
+        {
+            patients = (await _patientListRepository.FindAsync(c => c.FacilityId == facilityId, cancellationToken)).DistinctBy(p => p.PatientId).ToList();
```

Comment:
Might be more accurate to look into the patients stored in Report table for that particular report versus patient lists stored in Census table (cause those are modified with newer data)